core/src/Platforms/Android/SystemWebview/AuthenticationActivity.cs

```diff
@@ -212,54 +207,26 @@ private string GetChromePackageWithCustomTabSupport(Context context)
             }
 
             Intent customTabServiceIntent = new Intent(_customTabsServiceAction);
-            IList<ResolveInfo> resolveInfoList = context.PackageManager.QueryIntentServices(
-                    customTabServiceIntent, 0);
+
+            IList<ResolveInfo> resolveInfoListWithCustomTabs = context.PackageManager.QueryIntentServices(
+                    customTabServiceIntent, PackageInfoFlags.MatchAll);
 
             // queryIntentServices could return null or an empty list if no matching service existed.
-            if (resolveInfoList == null || resolveInfoList.Count == 0)
+            if (resolveInfoListWithCustomTabs == null || resolveInfoListWithCustomTabs.Count == 0)
```

Comment:
>Count [](start = 87, length = 5)

nit: collection.Any()

Reply:
done.

---
In reply to: [227736307](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1347#discussion_r227736307) [](ancestors = 227736307)